bin/resources/GameIndex.yaml

```diff
@@ -3036,7 +3054,7 @@ SCES-51533:
     roundSprite: 1 # Fixes vertical lines and minor ghosting.
     wrapGSMem: 1 # Fixes FMV when in progressive mode.
 SCES-51578:
-  name: "Network Access Disc [Original, v4.02 & v4.03]"
+  name: "Network Access Disc [v1.03 - v6.00]"
```

Comment:
Is this just all versions then? if so it's probably not worth mentioning the version at all, since it's kinda moot.

Reply:
There are two other serials, one for v0.02 and one for v1.02 : https://wiki.pcsx2.net/Network_Access_Disc